internal/cloudrunci/gcloud_test.go

```diff
@@ -27,21 +27,32 @@ func TestCreateIDToken(t *testing.T) {
 	// TODO assign to token
 	_, err := CreateIDToken("http://example.com")
 	if err != nil {
-		t.Errorf("CreateIDToken: %q", err)
+		t.Errorf("CreateIDToken: %v", err)
 	}
 
 	// validate token
 }
 
 func TestGcloud(t *testing.T) {
 	testutil.EndToEndTest(t)
-	out, err := gcloud("label", exec.Command(gcloudBin, "help"))
+	out, err := gcloudWithoutRetry("success", exec.Command(gcloudBin, "help"))
 	if err != nil {
-		t.Errorf("CreateIDToken: %q", err)
+		t.Errorf("gcloud: %v", err)
 	}
 
 	want := "gcloud - manage Google Cloud Platform resources and developer workflow"
 	if got := string(out); !strings.Contains(got, want) {
 		t.Errorf("gcloud: got (%s), want (%s)", got, want)
 	}
 }
+
+func TestGcloudRetry(t *testing.T) {
+	_, err := gcloud("failing", exec.Command("command-does-not-exist"))
```

Comment:
Can we hide the logs from this? It's a bit confusing to see "errors" in the logs when they're expected.

Reply:
Setting the logs to ioutil.Discard for the duration of this test.